include/flyft/mesh.h

```diff
@@ -19,22 +19,22 @@ class Mesh
          BoundaryType lower_bc,
          BoundaryType upper_bc);
 
-    std::shared_ptr<Mesh> slice(int start, int end) const;
+    std::shared_ptr<Mesh> slice(std::vector<int> start, std::vector<int> end) const;
```

Comment:
Make sure to pass all these vectors by const reference to avoid copies
```suggestion
    std::shared_ptr<Mesh> slice(const std::vector<int>& start, const std::vector<int>& end) const;
```